addon/-private/system/store.js

```diff
@@ -2408,60 +2414,34 @@ Store = Service.extend({
     assert("You need to pass a model name to the store's serializerFor method", isPresent(modelName));
     assert(`Passing classes to store.serializerFor has been removed. Please pass a dasherized string instead of ${Ember.inspect(modelName)}`, typeof modelName === 'string');
 
-    var fallbacks = [
-      'application',
-      this.adapterFor(modelName).get('defaultSerializer'),
-      '-default'
-    ];
+    let normalizedModelName = normalizeModelName(modelName);
 
-    var serializer = this.lookupSerializer(modelName, fallbacks);
-    return serializer;
-  },
-
-  /**
-    Retrieve a particular instance from the
-    container cache. If not found, creates it and
-    placing it in the cache.
-
-    Enabled a store to manage local instances of
-    adapters and serializers.
-
-    @method retrieveManagedInstance
-    @private
-    @param {String} modelName the object modelName
-    @param {String} name the object name
-    @param {Array} fallbacks the fallback objects to lookup if the lookup for modelName or 'application' fails
-    @return {Ember.Object}
-  */
-  retrieveManagedInstance(type, modelName, fallbacks) {
-    heimdall.increment(retrieveManagedInstance);
-    var normalizedModelName = normalizeModelName(modelName);
-
-    var instance = this._instanceCache.get(type, normalizedModelName, fallbacks);
-    set(instance, 'store', this);
-    return instance;
+    return this._instanceCache.get('serializer', normalizedModelName);
   },
 
   lookupAdapter(name) {
-    return this.retrieveManagedInstance('adapter', name, this.get('_adapterFallbacks'));
+    deprecate(`Use of lookupAdapter is deprecated, use adapterFor instead.`, {
+      id: 'ember-data:lookupAdapter',
+      since: '2.10'
```

Comment:
I believe `until` is the only option http://emberjs.com/api/#method_deprecate. Setting `since` should have no impact on the deprecation message.